addons/account_edi_ubl_cii/models/account_move_send.py

```diff
@@ -174,11 +154,12 @@ def _postprocess_document(self, invoice, prepared_data):
         writer = OdooPdfFileWriter()
         writer.cloneReaderDocumentRoot(reader)
 
-        writer.addAttachment(cii_xml['name'], cii_xml['raw'], subtype='text/xml')
+        writer.addAttachment('factur-x.xml', xml_facturx, subtype='text/xml')
 
         # PDF-A.
-        options = prepared_data['cii_xml_options']['options']
-        if options.get('facturx_pdfa') and not writer.is_pdfa:
+        if 'xml_options' in prepared_data \
+                and prepared_data['xml_options']['options'].get('facturx_pdfa') \
```

Comment:
wondering if we should keep this option; facturx_pdfa. or we can just check if 'facturx'

Reply:
good catch !